app/controllers/reviews_controller.rb

```diff
@@ -34,6 +32,19 @@ def update
     end
   end
 
+  def destroy
+    review
+    park
```

Comment:
Remove both `review` and `park` , you should use a callback to run this for the appropriate actions